package/.coveragerc

```diff
@@ -1,7 +1,7 @@
 [report]
-fail_under=100
-show_missing=True
-omit=*/plugin.py
+show_missing = True
+omit =
+    */integrations/kedro/data_loader.py  
```

Comment:
This abstracts away the data loading process in Kedro, e.g. `session.load_context` or `import pipelines` while providing backward-compat check. Testing this manually and with e2e test because unit testing here is basically testing Kedro.